exercises/clock/Clock.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I don't know an easy way to revert this change as the commit that contains it is mixed in with other changes :(

Reply:
Ok `git reset` is available on a per file basis... you learn something new everyday :)
`git reset <commit hash> <filename>`